app/src/main/java/com/github/se/polyfit/ui/screen/PostInfoScreen.kt

```diff
@@ -61,31 +60,32 @@ fun PostInfoScreenContent(
     padding: PaddingValues = PaddingValues(),
     viewPostViewModel: ViewPostViewModel = hiltViewModel(),
 ) {
-  val posts = viewPostViewModel.getAllPost().observeAsState(initial = listOf())
-  val isFetching by viewPostViewModel.isFetching.collectAsState()
 
   Scaffold(modifier = Modifier.padding(padding)) {
-    if (isFetching) {
+    val isFetching = viewPostViewModel.isFetching.value
+    val posts = viewPostViewModel.posts.collectAsState(initial = emptyList()).value
```

Comment:
This works well and makes usage of the viewModel, but a suggestion I would have is to declare it as
`val posts by viewPostViewModel.posts.collectAsState(emptyList())`